apps/server/src/shared/repo/user/user.repo.ts

```diff
@@ -6,7 +6,7 @@ import { ImportUser, Role, SchoolEntity, User } from '@shared/domain/entity';
 import { IFindOptions, SortOrder } from '@shared/domain/interface';
 import { Counted, EntityId, NameMatch } from '@shared/domain/types';
 import { BaseRepo } from '@shared/repo/base.repo';
-import { MongoPatterns } from '../mongo.patterns';
+import { MongoPatterns } from '@shared/repo';

```

Comment:
'@shared' should be used only outside of the shared folder, AFAIK. Use the local path instead here.